tests/petsc/64.cc

```diff
@@ -67,8 +70,18 @@ main(int argc, char **argv)
         const unsigned int n_mpi_processes = static_cast<unsigned int>(n_jobs);
         Assert(n_dofs % n_mpi_processes == 0, ExcInternalError());
         const unsigned int n_local_dofs = n_dofs / n_mpi_processes;
```

Comment:
You don't use `n_local_dofs` anywhere in this test. Is that intentional?

Reply:
No - that should be removed.